.github/workflows/ci.yml

```diff
@@ -17,6 +17,8 @@ jobs:
   build-libs-linux:
     name: build-libs-linux-${{ matrix.arch }} 😁 (${{ matrix.name }})
     runs-on: ${{ (matrix.arch == 'arm64' && 'github-arm64-2c-8gb') || 'ubuntu-22.04' }}
+    container:
```

Comment:
There is another `container` below: https://github.com/falcosecurity/libs/pull/2158/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR38